index.html

```diff
@@ -8,7 +8,41 @@
     <title>Document</title>
   </head>
   <body>
-    <section class="advantages">
+    <header class="header container">
+      <nav class="nav__panel">
+        <div class="nav__panel--logo" src="./images/Triphouse_logo.png" alt="Triphouse logo"></div>
```

Comment:
Your sprite must have svg format. Now it does not work. Also your background doesn't show

Reply:
Sorry, looks like I'd missed that moment. https://github.com/pavel-4/if-html/pull/5/commits/67a55bde72134abf89ed24dbe634e992936a29f0 will solve that issue.

Reply:
I have some problems with  your background image, it is not showiing, but in css file it is  looking ok.